### PR TITLE
fix: text in home page cards

### LINKED ISSUE #92: Fix the text in the home page cards
### Contact Details

_No response_

### What happened?

The text in the card should be centered, the font size is a bit small , need to increase that. Also, show ellipses `...` when the title overflows

image

### In which device are you seeing the problem on?

_No response_

### Relevant code

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by V9vek**
## Description

Fixed the text in the home page cards by making the text centered and adding ellipses when text gets long

## Issue Resolved

Fixes #92 

## Screenshots 

img2

img1

## Checklist

Please make sure to review the following before submitting your PR: 

- [x] I have read the [contribution guidelines](https://github.com/PEC-CSS/MovieDroid/blob/main/CONTRIBUTING.md).
- [x] I have read the code of conduct.
- [x] I have reviewed my submission in detail.

### CODE REVIEW

File: app/google-services.json [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
         {
-          "client_id": "312195271485-rd44fta4n39r4p8l7amp8n53lpdgeqdj.apps.googleusercontent.com",
+          "client_id": "849364734252-fbvq2nkoqehliodn28srg79mbsvf75d3.apps.googleusercontent.com",
+          "client_type": 1,
+          "android_info": {
+            "package_name": "com.pec_acm.moviedroid",
+            "certificate_hash": "c7db101352ad093949cf5513202f51dafb276a90"
+          }
+        },
+        {
```
Comment (ken1000minus7) [inline app/google-services.json]: You don't have to commit changes to this file, please remove these 😅

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#Wed Jan 26 14:26:41 IST 2022
+#Wed Oct 12 19:05:06 IST 2022
 distributionBase=GRADLE_USER_HOME
```
Comment (ken1000minus7) [inline gradle/wrapper/gradle-wrapper.properties]: This isn't required, please remove it 😇

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]
Great work @V9vek 😄 , a few changes are required, please go through them, once completed, your pr will be ready to be merged 😇

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (V9vek) [reply gradle/wrapper/gradle-wrapper.properties]: Sure

**Comment by V9vek** [comment]
I have reverted those changes to previous code @ken1000minus7 
please take a look now and do let me know 😊

**Review by ken1000minus7 — APPROVED** [review]
Lgtm, thanks for your contribution 😄

**Pull request merged**